pkg/clusteragent/admission/controllers/webhook/controller_v1beta1_test.go

```diff
@@ -92,9 +91,6 @@ func TestCreateWebhookV1beta1(t *testing.T) {
 }
 
 func TestUpdateOutdatedWebhookV1beta1(t *testing.T) {
-	if runtime.GOOS == "darwin" {
-		t.Skip("Skipping flaky test on macOS")
-	}
```

Comment:
❓ why are we removing these lines ?

Reply:
The explanation is in [this commit](https://github.com/DataDog/datadog-agent/pull/31022/commits/94dbcb7b6a95ff433e5eadab5fdf329fa3a10fa8). 
`SA4032 due to the file's build constraints, runtime.GOOS will never equal "darwin"`